gradle.properties

```
@@ -31,25 +31,25 @@ protobufVersion=3.14.0
 perfmarkVersion=0.23.0
 
 # Test Module Dependecy Versions
-stdlibGrpcVersion=1.1.0
-stdlibCryptoVersion=2.1.0
-stdlibIoVersion=1.1.0
-stdlibLogVersion=2.1.0
-stdlibHttpVersion=2.1.0
-stdlibJwtVersion=2.1.0
-stdlibUuidVersion=1.1.0
-stdlibOAuth2Version=2.1.0
-stdlibFileVersion=1.1.0
-stdlibRegexVersion=1.1.0
-stdlibTimeVersion=2.1.0
-stdlibOsVersion=1.1.0
-stdlibAuthVersion=2.1.0
-stdlibCacheVersion=3.1.0
-stdlibMimeVersion=2.1.0
-stdlibTaskVersion=2.1.0
-stdlibUrlVersion=2.1.0
-stdlibRandomVersion=1.1.0
-stdlibProtobufVersion=1.1.0
+stdlibGrpcVersion=1.2.0-20220112-015900-f3846a9
```

```suggestion
stdlibGrpcVersion=1.2.0-20220112-113500-19ba290
```

gRPC issue is fixed in this version.